File: challenge-331/hvukman/lua/331_p1.lua
-- http://lua-users.org/wiki/StringTrim
function Trim(s)
   return s:match'^()%s*$' and '' or s:match'^%s*(.*%S)'
end

function Last_length (X)
    local words = {}

    for word in Trim(X):gmatch("%w+") do
        table.insert(words, word)
    end

    local last_word = words[#words]
    print(#last_word)

end

Last_length("The Weekly Challenge")
Last_length("   Hello   World    ")
Last_length("Let's begin the fun")

File: challenge-331/hvukman/f#/331_p1.fs
// part 1
// Trim white space, split at space, take last and length
printf "part 1\n"
let lastword (x:string) = x.Trim(' ').Split(" ") |> Array.last |> String.length
printf "%i\n" (lastword "The Weekly Challenge")
printf "%i\n" (lastword " hello world    ")
printf "%i\n" (lastword "Let's begin the fun")
